src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10186,6 +10202,14 @@ namespace HVACVariableRefrigerantFlow {
         // Otherwise the coil needs to turn on. Get full load result
         PartLoadRatio = 1.0;
         this->CalcVRF_FluidTCtrl(VRFTUNum, FirstHVACIteration, PartLoadRatio, FullOutput, OnOffAirFlowRatio);
+        this->fanOutletT = Node(this->fanOutletNode).Temp;
```

Comment:
Here, when the TU operates the conditions resulting from PLR=1 (full flow to mimic what was done before) are captured and used in calculations for evap/cond temps.